src/components/FavorizeButton/FavorizeButton.spec.js

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import FavorizeButton from './FavorizeButton'
+
+describe('FavorizeButton', () => {
+  it('renders a button with an svg and text', () => {
+    render(
+      <FavorizeButton isFavorite={false} onToggleFavorite={jest.fn} id={1} />
+    )
+    expect(screen.getByText('Add to Favorites')).toBeInTheDocument()
+    expect(screen.getByTestId('svg')).toBeInTheDocument()
```

Comment:
You could be more precise here and look for the button and for the svg using `getByRole` / `getByTitle`.